tools/cldr-apps/src/main/java/org/unicode/cldr/web/Dashboard.java

```diff
@@ -1,26 +1,14 @@
 package org.unicode.cldr.web;
 
-import java.util.ArrayList;
-import java.util.EnumSet;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
Why the *? seems to be the best practice to give explicit imports.

Reply:
IntelliJ or VS Code did it automatically without my asking, sorry!

Reply:
ok. must have some threshhold, 'over 10 lines and we put a * in'. No objection…